src/v2/guide/index.md

```diff
@@ -393,6 +393,11 @@ You may have noticed that Vue components are very similar to **Custom Elements**
 
 2. Vue components provide important features that are not available in plain custom elements, most notably cross-component data flow, custom event communication and build tool integrations.
 
+Although Vue doesn't use custom elements internally, it has great interoperability when it comes to
+consuming or distributing as custom elements. According to <https://custom-elements-everywhere.com/#vue>,
```

Comment:
In order to keep the words concise while still maintaining the same information, I'd suggest something like:

> Although Vue doesn't use custom elements internally, it has [great interoperability](https://custom-elements-everywhere.com/#vue) when it comes to consuming or distributing as custom elements. Vue CLI also supports building Vue components that register themselves as native custom elements.